src/supertux/sector.cpp

```diff
@@ -1284,6 +1284,36 @@ Sector::get_height() const
   return height;
 }
 
+Size
+Sector::get_editor_size() const
+{
+  // Find the solid tilemap with the greatest surface
+  size_t max_surface = 0;
+  Size size;
+  for(auto& solids: solid_tilemaps) {
+    size_t surface = solids->get_width() * solids->get_height();
+    if (surface > max_surface) {
+      max_surface = surface;
+      size.width = solids->get_width();
+      size.height = solids->get_height();
+    }
+  }
+
+  return size;
+}
+
+void
+Sector::resize_sector(Size old_size, Size new_size)
+{
+  for(auto& object : gameobjects) {
+    TileMap* tilemap = dynamic_cast<TileMap*>(object.get());
+    if (tilemap && int(tilemap->get_width()) == old_size.width &&
```

Comment:
I am not sure if this is possible:
Instead of comparing width and height here, does the "Size" type have a == operator that we can use?